packages/sw-cache-expiration/src/lib/behavior.js

```diff
@@ -118,15 +118,78 @@ class Behavior {
 
   /**
    * A "lifecycle" callback that will be triggered automatically by the
-   * goog.runtimeCaching handlers when an entry is added to a cache.
+   * `goog.runtimeCaching` handlers when a `Response` is about to be returned
+   * from a [Cache](https://developer.mozilla.org/en-US/docs/Web/API/Cache) to
+   * the handler. It allows the `Response` to be inspected for freshness and
+   * prevents it from being used if the `Response`'s `Date:` header value is
+   * older than the configured `maxAgeSeconds`.
+   *
+   * Developers who are not using `goog.runtimeCaching` would normally not call
+   * this method directly; instead, [`isResponseFresh`](#isResponseFresh) can be
+   * used to perform the same freshness check.
```

Comment:
I won't consider this blocking. In the future, please try to use active voice instead of passive. For example, "instead, use `isResponseFresh` to perform the same freshness check.